packages/util-waiter/src/poller.ts

```diff
@@ -13,22 +18,19 @@ function exponentialBackoff(floor: number, ciel: number, attempt: number): numbe
  * @param stateChecker function that checks the acceptor states on each poll.
  */
 export const runPolling = async <T, S>(
-  params: WaiterOptions,
+  { minDelay, maxDelay }: WaiterOptions,
   client: T,
   input: S,
   acceptorChecks: (client: T, input: S) => Promise<WaiterResult>
 ): Promise<WaiterResult> => {
   let currentAttempt = 1;
-  let currentDelay = params.minDelay;
 
   while (true) {
-    await sleep(currentDelay);
+    await sleep(exponentialBackoff(minDelay, maxDelay, currentAttempt));
```

Comment:
nit: should this be renamed to jitterBackoff or something like that?

Reply:
It's just a variant of exponential backoff. I can rename it to `exponentialBackoffWithJitter()`

Reply:
But the name is just a little too long